apps/lido/app/src/components/ChangeELRewardsWithdrawalLimitSidePanel.js

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback } from 'react'
+import { Button, GU, SidePanel, Info } from '@aragon/ui'
+import * as yup from 'yup'
+import { Formik, Field } from 'formik'
+import TextField from './TextField'
+
+const initialValues = {
+  limit: '',
+}
+
+const validationSchema = yup.object().shape({
+  limit: yup
+    .number()
+    .positive()
+    .required()
+    .min(0)
+    .max(100)
+    .test(
+      'limit',
+      `Limit must be an integer or have 1 or 2 decimal places.`,
```

Comment:
The message may sound a bit confusing. 
I propose to rewrite smth like:
`The limit must be a number with up to 2 optional decimal places`

Reply:
Yeah, thought so too, I like yours better :)